GVRf/Sample/scene-objects/src/org/gearvrf/sample/sceneobjects/SceneObjectActivity.java

```diff
@@ -30,15 +37,87 @@
     private float yangle = 0;
     private float xangle = 0;
     private long lastDownTime = 0;
+    private WebView webView;
+    private Camera camera;
 
     /** Called when the activity is first created. */
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        mViewManager = new SampleViewManager();
+
+        createWebView();
+        createCameraView();
+
+        mViewManager = new SampleViewManager(this);
         setScript(mViewManager, "gvr_note4.xml");
     }
 
+    private void createWebView() {
+        webView = new WebView(this);
+        webView.setInitialScale(100);
+        webView.measure(2000, 1000);
+        webView.layout(0, 0, 2000, 1000);
+        WebSettings webSettings = webView.getSettings();
+        webSettings.setJavaScriptEnabled(true);
+        webView.loadUrl("http://gearvrf.org");
+        webView.setWebViewClient(new WebViewClient() {
+            @Override
+            public boolean shouldOverrideUrlLoading(WebView view, String url) {
+                view.loadUrl(url);
+                return true;
+            }
+        });
+    }
+
+    private boolean checkCameraHardware(Context context) {
+        if (context.getPackageManager().hasSystemFeature(
```

Comment:
heh, it's what you get when you read the google documentation on the camera :)  http://developer.android.com/guide/topics/media/camera.html

I changed it to the simple return you suggest in my latest commit.